ionic-template/slice.js

```diff
@@ -0,0 +1,39 @@
+import { createSlice } from '@reduxjs/toolkit'
+// import { selectorForSlice } from 'lp-redux-utils'
+// import * as apiActions from 'api-actions'
+// import { handleSuccess } from 'lp-redux-api'
+
+export const name = '%subsection%'
+const initialState = {}
+
+const slice = createSlice({
+  name,
+  initialState,
+  reducers: {
```

Comment:
I think the difference between `reducers` and `extraReducers` (**really bad name**, BTW. Nice going RTK) will initially trip folks up. It took me looking at this code and reading the documentation several times to understand what was going on. `reducers` are for locally-defined actions, essentially what used to be in the `actions.js` file. `extraReducers` are for externally-defined actions, like the API example shown here. I wonder if some comments should be added to this boilerplate to make that clear. This goes back to my `client-template` comment about how folks learn to use this new library.

Reply:
Yeah, it's a terrible name. The maintainer admitted that it probably should have been `externalReducers`

Reply:
I intend to follow up the launch with a dev committee presentation. Additionally, this topic is covered briefly in the Onboarding tutorial!